chpl/chpl-service/src/main/resources/log4j2-app.xml

```diff
@@ -17,6 +17,9 @@
                     interval="1" modulate="true" />
             </Policies>
         </RollingFile>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
```

Comment:
Are you changing the resources-dev, -staging, and -production log4j files also?

Reply:
No - these changes were just made to make my local debugging easier.  There was no intention to change how logging works in any other environment.